### ISSUE TITLE
Overlay events

### ISSUE THREAD

**Issue by acpmasquerade** [brightcove/videojs-overlay#33]
1. overlay-shown
2. overlay-hidden
3. overlay-ready

**Comment by misteroneill** [comment]
Is this a feature request? If so, it would be nice that were more clear.

Also, since this is open source, you are welcome to open a pull request!

**Comment by acpmasquerade** [comment]
Yes, it's a feature request. Also it's a very quick implementation. I just triggered respective events before the return statement in hide(), show() and createEl() functions.

However, I couldn't get it tested.
